src/buildCommon.js

```diff
@@ -636,10 +637,10 @@ const makeVList = function(params: VListParam, options: Options): DomSpan {
             }
 
             realChildren.push(childWrap);
-            currPos += elem.height + elem.depth;
+            currPos += utils.round(elem.height + elem.depth);
         }
-        minPos = Math.min(minPos, currPos);
-        maxPos = Math.max(maxPos, currPos);
+        minPos = utils.round(Math.min(minPos, currPos));
+        maxPos = utils.round(Math.max(maxPos, currPos));
```

Comment:
It seems rounding errors accumulate over the loop. Could you move the rounding out of the loop?